docs/guides/data/types-of-data.md

```diff
@@ -6,19 +6,48 @@
-| gVCF        | [Genomic VCF][gvcf-spec] files produced by [Microsoft Genomics Service][msgen]               | [Click here](#gvcf-files)        |
-| Somatic VCF | Curated list of somatic variants produced by the St. Jude somatic variant analysis pipeline | [Click here](#somatic-vcf-files) |
-| CNV | list of somatic copy number alterations produced by St. Jude CONSERTING pipeline | [Click here](#cnv-files) |
+| File Type   | Short Description                                                                                                 | Details                          |
+| ----------- | ----------------------------------------------------------------------------------------------------------------- | -------------------------------- |
+| BAM         | HG38 aligned BAM files produced by [Microsoft Genomics Service][msgen] (DNA-Seq) or STAR 2-pass mapping (RNA-Seq). | [Click here](#bam-files)         |
+| gVCF        | [Genomic VCF][gvcf-spec] files produced by [Microsoft Genomics Service][msgen].                                    | [Click here](#gvcf-files)        |
+| Somatic VCF | Curated list of somatic variants produced by the St. Jude somatic variant analysis pipeline.                       | [Click here](#somatic-vcf-files) |
+| CNV         | list of somatic copy number alterations produced by St. Jude CONSERTING pipeline.                                  | [Click here](#cnv-files)         |
 
 
 #### BAM files
 
-In St. Jude Cloud, we stored aligned sequence reads in the ubiquitous BAM file format. BAM files were produced by the [Microsoft Genomics Service][msgen] aligned to HG38 (GRCh38 no alt analysis set). For more information about how Microsoft Genomics produces BAM files or any other questions regarding data generation, please refer to [the official Microsoft Genomics whitepaper][msgen-whitepaper].
-
-For more information on SAM/BAM files, please refer to the [SAM/BAM specification][bam-spec]. 
+In St. Jude Cloud, we stored aligned sequence reads in the ubiquitous BAM file format. For more information on SAM/BAM files, please refer to the [SAM/BAM specification][bam-spec]. 
+
+##### Whole Genome and Whole Exome
+Whole Genome Sequence (WGS) and Whole Exome Sequence (WXS) BAM files were produced by the [Microsoft Genomics Service][msgen] aligned to HG38 (GRCh38 no alt analysis set). For more information about how Microsoft Genomics produces BAM files or any other questions regarding data generation, please refer to [the official Microsoft Genomics whitepaper][msgen-whitepaper].
+
+##### RNA-Seq
+
+RNA-Seq BAM files were produced by aligning the sequences to HG38 using `STAR` (v2.5.3a) 2-pass mapping followed by `Picard MarkDuplicates`. Below is the `STAR` command used during alignment. For more information about any of the parameters used, please refer to the [STAR manual][star-manual] for v2.5.3a.
+
+```bash
+    STAR \
+        --runThreadN $NUM_THREADS \
```

Comment:
It may be worth explaining the variables used in generating the command line. Possibly $GENOME_DIR is the only relevant one to specify the exact reference used.